lms/djangoapps/email_marketing/signals.py

```diff
@@ -148,9 +148,9 @@ def email_marketing_user_field_changed(sender, user=None, table=None, setting=No
         if not email_config.enabled:
             return
 
-        # perform update asynchronously
+        # perform update asynchronously, flag if activation
```

Comment:
#nitpick The comment is confusing. Can we be more descriptive? Like `set the activation flag when the user is marked as activated`?